src/frontend/src/elements/BackendLink.tsx

```diff
@@ -0,0 +1,6 @@
+import type { PropsWithChildren } from "react";
+
+
+export default function BackendLink(props: PropsWithChildren<JSX.IntrinsicElements["a"]>) {
```

Comment:
```suggestion
export default function Link(props: PropsWithChildren<JSX.IntrinsicElements["a"]>) {
```
Das ist ja ein normaler link. Der könnte auch direkt an github oder so gehen...
Wenn ein SPA link hinzukommt würde ich den liebr SpaLink nennen.